docs/parse.md

```diff
@@ -0,0 +1,47 @@
+# `Temporal.parse`
+
+The `Temporal.parse` function parses a full ISO string and divides it into its
+respective components, which can then be used to produce the corresponding
+Temporal objects.
+
+This function can be used to parse full valid ISO strings that represent invalid
+absolutes and salvage valid components from the string.
+
+## Syntax
```

Comment:
I'd prefer to keep the same style as the rest of the docs, putting the syntax in a h3:
```md
### Temporal.**parse**(input: string) : object
```
or
```md
### Temporal.**parse**(input: string) : { absolute: string, dateTime: string, etc... }
```

Reply:
So, regarding this and all other related comments, I noted that I was deviating from the rest of the documentation, but I figured it would be okay, since all other pages described a class and then further used the format that you're suggesting to describe a method *on that class*, while this is a first-class method on the `Temporal` global. Since I found nothing similar to it (first-class methods, should be treated like `Date`, for example, not like `Date.from`) in the current documentation, and since we wanted to go with MDN-style documentation anyway, I went ahead and borrowed this format from MDN. If you still feel that I should use the same format here, then let me know.

Reply:
That sounds fine.

Although in the processing script for this Markdown there is some magic in h3's that generates a permalink icon, a human-readable anchor, and bolds the function name to make it stand out. I'd still recommend putting the function signature in a h3 if possible, to take advantage of that. So the anchor name would become `#parse` instead of `#Temporal-parse-input-string-object`.

Alternatively, you could make that magic smarter so that it only does the magic if the heading looks like a function signature, and processes h2's and other headings as well. (I didn't get around to doing this yet.)

Reply:
I don't think we need markers in this tiny document, since it's for a single function anyway. Plus the whole link is just `parse.html`, which seems fine.

I added some code to make sure the special linking only applies to function signatures, though.